### DISCUSSION TITLE
Feature Request: Automatically configure PHP -> Servers mapping for Xdebug

### DISCUSSION THREAD

**Discussion by graham73may** [php-perfect/ddev-intellij-plugin#73] [Ideas]
Really love that this plugin is setting up things like the data sources. It would be great if it could also set up the paths inside Preferences -> PHP -> Servers for xdebugging. 

E.g. this part of the ddev docs: 
https://ddev.readthedocs.io/en/stable/users/step-debugging/#phpstorm-and-command-line-debugging

Thanks!

**Comment by nico-loeber** [comment]
Hey @graham73may, I'm afraid you're too late! ;)
This feature is already implement in https://github.com/php-perfect/ddev-intellij-plugin/pull/57 the related issue was https://github.com/php-perfect/ddev-intellij-plugin/issues/55.

This feature made it not into the beta as I've taken a faux approach first. It's already merged in master, in case you like to test it. 
Also the stable version 1.0 will be released soon, probably on 30th of June.

**Reply by graham73may** [reply]
Oh that's awesome! Sorry I didn't realise, I'm using the beta download from the IntelliJ site. (Saw a link to this plugin in their newsletter and couldn't resist!) 

Look forward to 1.0 :) 

Thanks!

**Reply by nico-loeber** [reply]
No problem at all!
